src/vector/jitsi/index.ts

```diff
@@ -70,7 +70,7 @@ let meetApi: any; // JitsiMeetExternalAPI
         const theme = qsParam('theme', true);
 
         if (theme) {
-            document.body.classList.add(`theme-${theme}`);
+            document.body.classList.add(`theme-${theme.replace(" ", "_")}`);
```

Comment:
Doesn't the React side change mean there aren't spaces in the theme name any more...? Or is this meant as an extra precaution?

Reply:
Extra precaution, better than it exploding